packages/spor-datepicker-react/src/DateField.tsx

```diff
@@ -54,6 +55,7 @@ export function DateField(props: DateFieldProps) {
           <DateSegment key={i} segment={segment} state={state} />
         ))}
       </Flex>
+      <input type="hidden" value={state.value?.toString()} name={props.name} />
```

Comment:
Merk - her returnerer jeg ISO-datoen - i formatet `2022-12-24` for julaften f.eks. Er det ønsket format?

Reply:
Jeg tenker at ISO er smart. Så får folk konvertere selv hvis man ønsker et annet format